src/user.ts

```diff
@@ -0,0 +1,50 @@
+import { fetch } from "undici";
+import { BOT_DEVICE_ID, KEYV_BOT_STORAGE } from "./env";
+import { KeyvStorageProvider } from "./storage";
+import { LogService } from "matrix-bot-sdk";
+import { API_URL } from "./env.js";
+
+export interface UserResponse {
```

Comment:
Could we rename this interface into Identity? So its consistent with the Directory service naming and all the other namings in our system?